ghostwriter-jdk-v7/src/main/java/io/ghostwriter/openjdk/v7/GhostWriterAnnotationProcessor.java

```diff
@@ -50,8 +50,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             }
         }
 
-        final boolean doClaimAnnotations = false;
-        return doClaimAnnotations;
+        return false;
```

Comment:
Hi @hosszubalazs,

Sorry for the late reply. You actually pinpointed an issue. The process function in case of GW should always return false, since GW works without annotations. Can you do a proposed solution based on this assumption?